apps/common/views/accessibility.html

```diff
@@ -1,114 +1,119 @@
+          criterion 4.1.2 (Name, Role Value)</li>
+        <li>the guidance read out on how to correctly populate date fields is incorrect - this does not meet WCAG success
+          criterion 3.3.2 (Labels and Instructions)</li>
+        <li>the guidance read out on how to navigate typeaheads does not apply to all device types - this does not meet WCAG
+          success criterion 3.3.2 (Labels and Instructions)</li>
+        <li>not all fields have legend text - this does not meet WCAG success criterion 1.3.1 (Info and Relationships)</li>
+      </ul>
+
+      <p><strong>Non-accessible content found in ‘Report a lost or stolen BRP’</strong></p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>date fields do not use unique IDs, which may make it harder for assistive technologies to accurately interpret
+          text - this does not meet WCAG success criterion 4.1.1 (Parsing)</li>
+      </ul>
+
+      <p><strong>Non-accessible content found in ‘My BRP has not been delivered’</strong></p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>the link text to contact UK Visas and Immgration does not clearly communicate the link’s purpose - this does not
+          meet WCAG success criterion 2.4.4 (Link Purpose)</li>
+      </ul>
+
+      <p><strong>Non-accessible content found in ‘Nominate someone to collect your BRP’</strong></p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>some radio buttons do not have unique IDs, which may make it harder for assistive technologies to accurately
+          interpret text - this does not meet WCAG success criterion 4.1.1 (Parsing)</li>
+        <li>date fields have incorrect aria labels - this does not meet WCAG 1.3.1 (Info and relationships) and WCAG success
+          criterion 4.1.2 (Name, Role, Value)</li>
+        <li>on one page, an option is read out that does not exist - this does not meet WCAG success criterion 3.3.2 (Labels
+          and Instructions)</li>
+      </ul>
+
+      <p>We plan to fix all of these issues by no later than 23 March 2021.</p>
+
+      <h3 class="govuk-heading-m">Disproportionate burden</h3>
+      <p>Not applicable.</p>
+
+      <h3 class="govuk-heading-m">Content that’s not within the scope of the accessibility regulations</h3>
+      <p>Not applicable.</p>
+
+      <h2 class="govuk-heading-m">Preparation of this accessibility statement</h2>
+      <p>This statement was prepared on 23 September 2020. It was last reviewed on 23 September 2020.</p>
+      <p>This service was last tested on 19 September 2020. The test was carried out by Marvell Consulting Ltd.</p>
+      <p>We tested every page of all 4 services using automated, semi-automated and manual testing. All journeys were tested
+        using a keyboard or screen reader. Selected journeys were carried out using voice commands.</p>
+    </div>
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
that must have taken ages to write

Reply:
For whoever wrote it out first probably! 😅 I'm actually wondering now whether this would be better in a markdown file?

Reply:
some of the folks have complained that markdown doesn't keep the CSS that's needed for govuk so all good